src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1056,6 +1057,78 @@ default MessageAction editMessage(@Nonnull MessageEmbed newContent)
     @CheckReturnValue
     MessageAction editMessageEmbeds(@Nonnull Collection<? extends MessageEmbed> embeds);
 
+    /**
+     * Edits this Message's ActionRows to the provided {@link net.dv8tion.jda.api.interactions.components.ActionRow ActionRows}.
+     * <br><b>Messages can only be edited by the account that sent them!</b>.
+     *
+     * <p>This message instance will not be updated by this operation, please use the response message instead.
+     *
+     * <p>The following {@link net.dv8tion.jda.api.requests.ErrorResponse ErrorResponses} are possible:
+     * <ul>
+     *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#MISSING_ACCESS MISSING_ACCESS}
+     *     <br>The edit was attempted after the account lost access to the {@link net.dv8tion.jda.api.entities.Guild Guild}
+     *         typically due to being kicked or removed.</li>
+     *
+     *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#MISSING_PERMISSIONS MISSING_PERMISSIONS}
+     *     <br>The edit was attempted after the account lost {@link net.dv8tion.jda.api.Permission#MESSAGE_WRITE Permission.MESSAGE_WRITE} in
+     *         the {@link net.dv8tion.jda.api.entities.TextChannel TextChannel}.</li>
+     *
+     *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#UNKNOWN_MESSAGE UNKNOWN_MESSAGE}
+     *     <br>The edit was attempted after the Message had been deleted.</li>
+     * </ul>
+     *
+     * @param  actionRows
```

Comment:
These do not have to be action rows, there will most certainly be more layouts than that. You should only refer to them as component layouts and name action row as an example.